nginx/config-files/hydroshare-ssl-nginx-ingress.conf.template

```diff
@@ -0,0 +1,47 @@
+# template for deploying front-end switch as nginx instance. 
+# This does almost no work locally, and instead reroutes all requests 
+# to the value of the template variable CURRENT_VM. 
+
+upstream backend {
+   server CURRENT_VM;
+}
+
+server {
+    listen          80;
+    server_name     FQDN_OR_IP;
+    rewrite ^/(.*)  https://FQDN_OR_IP/$1 permanent;
+}
+
+server {
+    listen          443 ssl;
+    server_name     FQDN_OR_IP;
+    root            /system/static/; 
+
+    access_log      /var/log/nginx/access.log combined;
+    error_log       /var/log/nginx/error.log debug;
+
+    sendfile off;
+    proxy_request_buffering off; 
+    proxy_buffering off; 
+
+    charset         utf-8;
+    client_max_body_size 4096m;  # limit uploads to 4 GiB. 
+
+    ssl_certificate         /hs-certs/SSL_CERT_FILE;
+    ssl_certificate_key     /hs-certs/SSL_KEY_FILE;
+
+    location / {
+        if (-f $document_root/maintenance_on.html) {
+            return 503;  # invoke error handling as defined below
+        }
+	proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+	proxy_pass https://backend;  # allows extra backend servers
```

Comment:
This should be updated to `proxy_pass https://CURRENT_VM;

Also, formatting is off with the number of spaces for indentation`

Reply:
Done.